src/app/features/notification/NotificationTimePipe.ts

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
@sleidig, Please share your suggestion on this, I have created this file for the notification time. Is this the correct way to implement it, or are there any changes you would recommend?